bin/python/ripple/util/Sign.py

```diff
@@ -22,9 +23,9 @@
     check <key>
         Check an existing key for validity.
 
-    sign <sequence> <validator-public> <master-secret>
+    sign <sequence> <validator-public-key> <validator-private-key> <master-secret>
         Create a new signed manifest with the given sequence
-        number, validator public key, and master secret key.
+        number, validation public key, and master public key.
```

Comment:
No problem.  The only reason I knew the `manifest-tool-guide` was there was from my previous puzzlement and fumbling with manifests.  And whichever key you need to use is fine with me.  We just need to be able to keep the names straight enough that a user can follow the instructions once every 9 months to 3 years when they need to change the ephemeral key.  Thanks for your understanding.